_posts/2019-07-31-quarkus-0-20.0-released.adoc

```diff
@@ -0,0 +1,112 @@
+====
+
+=== Improved integration of Hibernate Validator with CDI
+
+Hibernate Validator is now fully integrated with CDI in Quarkus. You can even tweak the `ValidatorFactory` just by declaring beans in your applications. Just have a look at link:/guides/validation-guide#going-further[the updated documentation] to learn more about it.
+
+=== New MicroProfile REST Client implementation
+
+In this release we've switched from using SmallRye REST Client to RESTEasy as an implementation of MicroProfile REST Client. As the MicroProfile REST Client builds on top of functionality that is already present within the JAX-RS Client and the RESTEasy Client, it made sense for us to combine all the clients into the RESTEasy project.
+
+This change also brings the REST Client implementation in Quarkus up to date with the latest MicroProfile REST Client specification, version 1.3, that is part of the latest MicroProfile 3.0 platform release. Some of the new features in the release include support for SSL configuration of a client, making JSON the default `MediaType` if there is not one specified by `@Produces` or `@Consumes`, and easier configuration with config keys to enable setting common configuration across multiple client interfaces.
+
+=== Configuring a per method transaction timeout
+
+You can now configure the transaction timeout of a specific transactional method using the `@TransactionConfiguration` annotation. Please refer to link:transaction-guide#transaction-configuration[our Transaction guide] for more information.
+
+=== Syslog support for logging
+
+Quarkus now supports shipping the logs to Syslog. Please refer to link:/guides/logging-guide#syslog-configuration[our updated logging documentation].
+
+=== New extensions
+
+==== Jackson extensions
+
+Quarkus loves standards.
+That's why we started by supporting JSON-B as our JSON serialization library.
+
+We had a lot of users asking for Jackson support and, while you could use Jackson with Quarkus, it wasn't as easy as for JSON-B.
+
+We just added two new extensions to make Jackson a first class citizen in Quarkus:
+
+ * `quarkus-jackson` for standalone usage of Jackson
+ * `quarkus-resteasy-jackson` to use Jackson for your REST services
+
+==== OAuth2 extension
+
+The new Security OAuth2 extension allows to implement an authentication mechanism based on OAuth2 opaque tokens. 
+It complements the existing JWT extension by allowing to authenticate a user based on a call to an introspection endpoint instead of decoding the token. 
+An opaque token is a token with no information encoded in it, so it needs an external call to gather the user information, this external call is made against what is called an introspection endpoint.
+
+Behind the scene, it uses Elytron and the same annotations as all the other security extensions, so it's as easy to use as the other authentication mechanism that Quarkus supports, and a user can easily switch between them if needed.
```

Comment:
Should it be `Behind the scenes`?

Reply:
Also `it uses Elytron` might be better as `Elytron is used`

Reply:
Fixed the first one. As for the second one, it doesn't work with the rest of the sentence so I kept it as is.

Reply:
:+1: